### ISSUE TITLE
`panic` is not detected if multiple arguments are provided

### ISSUE THREAD

**Issue by wawel37** [software-mansion/cairo-lint#273]
### Problem

if using panic like 
```cairo
panic!("panicing with {}", 5);
```

we doesn't detect it and ignore it

### Scarb Version

```text
2.11.3
```

### Notes

If panic is used with multiple arguments, it's being expanded by compiler to a block rather to a single function call
